index.ts

```diff
@@ -18,10 +18,7 @@ type CacheStorage<KeyType, ValueType> = {
 	clear?: () => void;
 };
 
-export type Options<
-	FunctionToMemoize extends AnyFunction,
-	CacheKeyType,
-> = {
+export type Options<FunctionToMemoize extends AnyFunction, CacheKeyType> = {
```

Comment:
Don't make unrelated changes. That makes it hard to review.

Reply:
Got it, I think my linter got to that one

Reply:
What do you have your linter line width set to?